listen.py

```diff
@@ -38,7 +38,7 @@ def from_json(cls, j):
             user_name=j.get('user_name', ""),
             timestamp=datetime.utcfromtimestamp(float(j['listened_at'])),
             artist_msid=j['track_metadata']['additional_info'].get('artist_msid'),
-            album_msid=j['track_metadata']['additional_info'].get('album_msid'),
```

Comment:
How complex is to to replace `album_*` with `release_*` everywhere in the `Listen` class and places that use it?